static/app/views/performance/landing/widgets/components/accordion.tsx

```diff
@@ -0,0 +1,91 @@
+import {ReactNode} from 'react';
+import styled from '@emotion/styled';
+
+import DropdownButton from 'sentry/components/dropdownButton';
+import space from 'sentry/styles/space';
+
+type Props = {
+  content: ReactNode;
+  items: (() => ReactNode)[];
+  selectedIndex: number;
+  setSelectedIndex: (index: number) => void;
```

Comment:
Also a nit, but in accordion parlance, "selected" is usually replaced with "expanded", and "collapsed". That would match the ARIA roles (if we ever add those)

Reply:
ah, nice! expanded makes more sense for sure 👍